README.md

```diff
@@ -1,4 +1,77 @@
 # CSharp-From-Zero-To-Hero
-[![Build Status](https://dev.azure.com/almantusk/From-Zero-To-Hero/_apis/build/status/Almantask.CSharp-From-Zero-To-Hero?branchName=master)](https://dev.azure.com/almantusk/From-Zero-To-Hero/_build/latest?definitionId=5&branchName=master)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a358adf21c2442cd91a4827a50213ff1)](https://www.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Almantask/CSharp-From-Zero-To-Hero&amp;utm_campaign=Badge_Grade)  
-C# boot camp
+[![Build Status](https://dev.azure.com/almantusk/From-Zero-To-Hero/_apis/build/status/csinn.CSharp-From-Zero-To-Hero?branchName=master)](https://dev.azure.com/almantusk/From-Zero-To-Hero/_build/latest?definitionId=6&branchName=master)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a358adf21c2442cd91a4827a50213ff1)](https://www.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=csinn/CSharp-From-Zero-To-Hero&amp;utm_campaign=Badge_Grade)
+  # The vision
+"Programming is hard". Yes.But not harder than running a marathon for a person has never run. It's not harder than 
+building a house if you never built one. Programming is hard only until you practice it (like any other skill). I would like to invite you to learn programming and C# following this course to practice programming and ignite passion for finding little miracles in code every day 🙂
+  
+  # Requirements
+  Visual Studio Community 2019 or similar with Web Development and desktop development workloads.  
+
+  Patience and persistence  
+  Git installed  
+  Github account  
+  
+  # Homework 
+  Homework will be announced at the end of every lesson and placed in this repository base on lesson name. There will be 2 branches after every lesson:  
+  ```
+  Lesson/LessonName  
+  Homework/LessonName  
+  ```
+  ## Git
+  Fork this repository (Homework/LessonName). 
+  The fork target is called upstream branch. For every chapter, you will need to keep updating upstream branch. 
+  How to work with forked repository in Visual Studio Teams:  
+  http://doc.fireflymigration.com/working-with-github-fork-in-visual-studio.html
+  
+  ## How to complete homework? 
+  ### Implementation of homework
+  For the first chapters, homework branch will come with 2 projects:
+  ```
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[fenced-code-flag] Missing code-language flag](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4835315)